crates/modelardb_server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -739,7 +750,10 @@ impl PhysicalExpr for ModelSumPhysicalExpr {
 /// models.
 #[derive(Debug)]
 struct ModelSumAccumulator {
+    /// Current sum stored by the [`Accumulator`].
     sum: f32,
+    /// Current count stored by the [`Accumulator`]. It is stored so it can be passed to
+    /// [SumAccumulator](repository/datafusion/physical-expr/src/aggregate/sum.rs).
```

Comment:
is it not necessary to have the full link to the GitHub repository?

Reply:
For some reason only `MinAccumulator`, `MaxAccumulator`, and `AvgAccumulator` are public, so as far as I know, I cannot link to `SumAccumulator` through doc comments.